### ISSUE TITLE
 Keyboard focus is going to top of the page after activating the 'Copy access token' icon button.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1370]
**Test Environment:**
OS Version: Windows 11 Build 22504.1010
Browser: Edge Dev Version 98.0.1093.6 (Official build) dev (64-bit)
URL: Graph Explorer - Microsoft Graph
Screen Reader: Narrator

**Repro Steps:**

1. Open above URL in Edge browser.
2. Login with valid credential.
3. Navigate to the 'Action token' button using tab key and activate it using enter key.
4. Now, navigate to the 'Copy access token' icon button using tab key and activate it.
5. Now navigate with tab key.
6. Observe if focus goes to the top of the page.

**Actual Result:**
Keyboard focus is going to top of the page after activating the 'Copy access token' icon button.

**Expected Result:**
Keyboard focus should be present on the 'Copy access token' button after activating the button.

**User Impact:** 
Keyboard users will be impacted as they will have to extra navigate to reach the specified control.
 
**MAS Reference:**
[Focus Order - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.04.03)
